bundles/org.openhab.binding.goecharger/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -181,13 +200,21 @@
 		<description>Specifies the max amps that can be charged with that cable</description>
 		<state pattern="%d %unit%" readOnly="true"/>
 	</channel-type>
+	<channel-type id="tmpT2p">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature type 2 port</label>
+		<description>Temperature of the Go-eCharger on the type 2 port</description>
+		<state pattern="%d %unit%" readOnly="true"/>
+		<category>Temperature</category>
+	</channel-type>
 	<channel-type id="tmp">
 		<item-type>Number:Temperature</item-type>
-		<label>Temperature</label>
-		<description>Temperature of the Go-eCharger</description>
+		<label>Temperature circuit board</label>
+		<description>Temperature of the Go-eCharger on circuit board</description>
 		<state pattern="%d %unit%" readOnly="true"/>
+		<category>Temperature</category>
 	</channel-type>
-	<channel-type id="fmw">
+	<channel-type id="fwv">
```

Comment:
Is this rename by purpose or left by mistake? Now that the other channel type id's are renamed back.